### DISCUSSION TITLE
Scaling Objects

### DISCUSSION THREAD

**Discussion by SSNTails** [jrouwe/JoltPhysics#1046] [Q&A] [answered]
Think of something that gets hit by a shrink ray. Is it absolutely necessary to wrap the shape in a ScaledShape, or is there a simpler way to change the shape dimensions if I don't care about collisions during growth/shrink?

**Comment by jrouwe** [comment, marked as answer]
ScaledShape is the only way. If you repeatedly change scale, I would make sure that you don't create a very long chain of ScaledShapes, just get the inner shape and wrap it with a new ScaledShape.
